build/webpack.base.js

```diff
@@ -4,7 +4,7 @@ var projectRoot = path.resolve(__dirname, '../')
 
 module.exports = {
   externals: {
-    uikit: 'UIkit'
+    uikit: 'uikit'
```

Comment:
Uikit is globally registered under `UIkit` space name. This change is wrong, the others seems ok.